pkg/chunk/table_client.go

```diff
@@ -16,15 +16,28 @@ type TableDesc struct {
 	ProvisionedRead  int64
 	ProvisionedWrite int64
 	Tags             Tags
+
+	WriteScaleEnabled     bool
+	WriteScaleRoleARN     string
+	WriteScaleMinCapacity int64
+	WriteScaleMaxCapacity int64
+	WriteScaleOutCooldown int64
+	WriteScaleInCooldown  int64
+	WriteScaleTargetValue float64
```

Comment:
This looks the same as what's in schema_config.go. Why not split these fields into a separate struct type?

Then `AutoScalingEquals` can become `Equals` on that object (and maybe even the implementation could be replaced with a call to `DeepEquals`)